components/ai_chat/core/browser/utils.h

```diff
@@ -34,6 +36,17 @@ void GetOCRText(const SkBitmap& image, GetOCRTextCallback callback);
 
 const std::string& GetActionTypeQuestion(mojom::ActionType action_type);
 
+// Returns the maximum length of the page content in characters.
+uint32_t GetMaxPageContentLength(const mojom::Model& model);
+uint32_t GetMaxPageContentLength(std::optional<uint32_t> context_size);
```

Comment:
why is the version of this that operates on a `uint32_t` public? `uint32_t GetMaxPageContentLength(const mojom::Model& model);` I guess is a reasonable workaround to the lack of an actual class with its own internal validation, etc..., but the `unint32_t` method here is an internal detail that doesn't need to be in the header